.github/dependabot.yml

```
@@ -13,6 +13,13 @@ updates:
     directory: "/frontend"
     schedule:
       interval: "weekly"
+    groups:
+      angular:
+        patterns:
+        - "@angular*"
+        update-types:
+        - "minor"
+        - "patch"
```

```suggestion
        update-types:
        - "major"
        - "minor"
        - "patch"
```

I'd say it makes sense to get notified about new major versions too.

---

I'm not that familiar with Angular's upgrade scheme. If you do not expect everything to break on a major version upgrade, then this change is fine.

---

I expect breaking changes, but dependabot doesn't auto-update on main. It just opens a PR so that we can looks into it.